### ISSUE TITLE
The reason for low performance, vocabs in train_dataset and test_dataset are not the same

### ISSUE THREAD

**Issue by zhanxlin** [minqukanq/hierarchical-multi-label-text-classification#6]
Maybe I found the reason for low performance, vocabs in train_dataset and test_dataset are not the same, that means the same token may have different index.
Can help check if right?

**Comment by minqukanq** [comment]
The data in the "/data" folder is only used to show the format of the data, not the data for training.
Please check /data/README.md.

**Comment by zhanxlin** [comment]
> 

I know it, but for the original dataset and for my own dataset, the performances are both too low. And I spent much time to find the problem. Now I think the reason is the train_dataset and the test_dataset use two different vocab_to_int. I fix this bug and then the performance grows up.

**Comment by minqukanq** [comment]
Wow,
Could you please pull requests the code if possible?

**Comment by minqukanq** [comment]
#7
